### PR TITLE
Set model attributes for POT model inside optimize function

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
Attributes should be set to ModelEnity to SC have information about exported model.

### CODE REVIEW

**Review by alexsu52 — COMMENTED** [review]
Could add a short description of why these changes are needed?

**Comment by MaximProshin** [comment]
@druzhkov-paul, please prioritize it. The changes seem to be simple so it shouldn't take a lot of time.

**Review by druzhkov-paul — APPROVED** [review]

File: tools/sc_sample.py [hunk 1]
```diff
@@ -154,13 +154,6 @@ def main(args):
         optimized_model = ModelEntity(
             dataset,
             environment.get_model_configuration(),
-            optimization_type=ModelOptimizationType.POT,
-            optimization_methods=OptimizationMethod.QUANTIZATION,
```
Comment (sovrasov) [inline tools/sc_sample.py:158]: import of OptimizationMethod is not used in this file anymore and should be removed.

Comment (AlexanderDokuchaev) [reply tools/sc_sample.py:158]: removed

**Review by sovrasov — APPROVED** [review]

**Pull request merged**